fontforge/splineutil2.c

```diff
@@ -2911,7 +2911,7 @@ void SplineSetAddInflections(SplineChar *sc, SplineSet *ss, int anysel) {
     Spline *s, *first; 
     first = NULL;
     for ( s = ss->first->next; s!=NULL && s!=first; s = s->to->next ) {
-	    if ( !anysel || s->from->selected && s->to->selected )
+	    if ( !anysel || (s->from->selected && s->to->selected) )
```

Comment:
@linusromer just confirming that this is the intended behaviour?

Reply:
Yes, either nothing is selected or both endpoints are selected. I thought that `a || b && c` is equivalent to `a || (b && c)` but I may be misled by other programming languages.